app/controllers/api/base_controller/renderer.rb

```diff
@@ -616,6 +616,18 @@ def determine_extra_cols(klass)
           attr_name.to_sym
         end
       end
+
+      def status_from_resource(resource)
+        return :ok unless resource.kind_of?(Hash)
+
+        if resource.key?(:success)
+          return resource[:success] ? :ok : :not_acceptable
+        end
+        results = resource["results"]
+        return :ok unless results.kind_of?(Array) && results.size == 1
+        return :ok unless results.first.kind_of?(Hash)
+        !results.first.key?(:success) || results.first[:success] ? :ok : :not_acceptables
```

Comment:
`: not_acceptables` is the plural a typo?

Reply:
totally - thanks